src/test/scala/uk/gov/nationalarchives/dp/client/testUtils/ValidateXmlAgainstXsd.scala

```diff
@@ -0,0 +1,36 @@
+package uk.gov.nationalarchives.dp.client.testUtils
```

Comment:
Did you mean to put this file in the test package? If the calling clients need to use it they won't be able to from here.